packages/create-astro/src/index.ts

```diff
@@ -52,12 +54,14 @@ export async function main() {
     },
   ]);
 
-  const emitter = degit(`snowpackjs/astro/examples/${options.template}`, {
+  const hash = args.commit ? `#${args.commit}` : '';
```

Comment:
Adds a `--commit [hash]` flag to `create-astro`. This was necessary so we’re pulling PR commit hashes rather than `main`.